Taskfile.yml

```diff
@@ -0,0 +1,33 @@
+version: '3'
+
+dotenv: ['.env']
+
+tasks:
```

Comment:
Consider grouping related tasks under a `group` functionality if your task runner supports it, which can help organize tasks logically and improve maintainability.